apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/model/ChatModel.kt

```diff
@@ -1762,6 +1811,31 @@ data class GroupInfo (
   val canModerate: Boolean
     get() = membership.memberRole >= GroupMemberRole.Moderator && membership.memberActive
 
+  fun nextNotificationMode(from: MsgFilter): ChatNtfs {
+    val nextNtfMode = when (from) {
+      All -> Mentions
+      Mentions -> None
+      None -> All
+    }
+    return ChatNtfs(nextNtfMode, notificationModeText(nextNtfMode), notificationModeIcon(nextNtfMode, false), notificationModeIcon(nextNtfMode, true))
+  }
+
+  private fun notificationModeText(m: MsgFilter): StringResource {
+    return when (m) {
+      All -> MR.strings.unmute_chat
+      Mentions -> MR.strings.mute_chat
+      None -> MR.strings.mute_all_chat
+    }
+  }
+
+  private fun notificationModeIcon(m: MsgFilter, filled: Boolean): ImageResource {
+    return when (m) {
+      None -> (if (filled) MR.images.ic_notifications_off_filled else MR.images.ic_notifications_off)
+      All -> MR.images.ic_notifications
+      Mentions -> if (filled) MR.images.ic_notifications_mentions_filled else MR.images.ic_notifications_mentions
```

Comment:
please name the icon file the same as on Material Icons website. We always preserve original name in order to know the source icon to be able to modify it if needed or to find a similar one